_includes/layouts/base.njk

```diff
@@ -5,11 +5,26 @@
 
 	<body>
 
-		<header>
-			<em>
-				<a href="{{ '/' | url }}">{{ metadata.title }}</a>
-			</em>
-			{% include "components/nav.njk" %}
+		<header class="header-main">
+			<div class="container container--wide container--flex">
+				<div class="header-main__logo">
+					<a href="{{ '/' | url }}">
+						<img src="/static/img/nlc-full-logoX600.png" alt="{{ metadata.title }}" />
+					</a>
+				</div>
+				<div class="header-main__nav">
+					{% include "components/nav.njk" %}
+				</div>
+				<div class="header-main__menu js-closed">
```

Comment:
This needs to be an element that's interactive so it can be focused on by keyboard users. I think semantically correct element is <button>: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button

Reply:
We need to use aria attributes to communicate the relationship between the toggle button and the menu, which is aria-expanded and aria-controls. There's a good example here: https://www.a11ymatters.com/pattern/mobile-nav/#using-aria-attributes-for-the-toggle-element

Reply:
Ah yes ok, makes sense in both cases. Amended markup and added aria-expanded: true/false functionality.